code/CMakeLists.txt

```diff
@@ -71,18 +71,24 @@ if(BuildSPEngine OR BuildJK2SPEngine)
 	endif()
 
 	if(UseInternalSDL2)
-		if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+		if(Architecture STREQUAL "x86")
 			set(SPEngineLibraries
 				${SPEngineLibraries}
 				${OpenJKLibDir}/SDL2/lib/x86/SDL2main.lib
 				${OpenJKLibDir}/SDL2/lib/x86/SDL2.lib
 				)
-		else()
+		elseif(Architecture STREQUAL "x86_64")
 			set(SPEngineLibraries
 				${SPEngineLibraries}
 				${OpenJKLibDir}/SDL2/lib/x64/SDL2main.lib
 				${OpenJKLibDir}/SDL2/lib/x64/SDL2.lib
 				)
+		elseif(Architecture STREQUAL "arm64")
+			set(SPEngineLibraries
+				${SPEngineLibraries}
+				${VcpkgDir}/installed/arm64-windows/lib/manual-link/SDL2main.lib
+				${VcpkgDir}/installed/arm64-windows/lib/SDL2.lib
```

Comment:
I don't love having vendored versions for everything _except_ arm64-windows.
I think my preference would be vendored versions for all, with optional sourcing from (system) package manager. I also don't love how much bloat and churn that entails. Maybe arm64-windows is a special-enough case to push for vcpkg.

This isn't a blocker, just provoking discussion so we can form a stance here intentionally.